src/main/java/perosnal/spotifymatcher/security/JwtTokenValidator.java

```diff
@@ -0,0 +1,64 @@
+package perosnal.spotifymatcher.security;
+
+import io.jsonwebtoken.Claims;
+import io.jsonwebtoken.Jws;
+import io.jsonwebtoken.Jwts;
+import lombok.AllArgsConstructor;
+import lombok.NoArgsConstructor;
+import lombok.SneakyThrows;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+import perosnal.spotifymatcher.util.HttpRequestSender;
+
+import java.security.KeyFactory;
+import java.security.PublicKey;
+import java.security.spec.X509EncodedKeySpec;
+import java.util.Base64;
+
+@Component
+@AllArgsConstructor
+@NoArgsConstructor
+public class JwtTokenValidator {
+
+    @Autowired
+    private HttpRequestSender httpRequestSender;
+
+    @Value("${auth.server.url}")
+    private String AUTH_SERVER_URL;
+
+
+    public void validateJwt(String jwt) {
+        Jwts.parserBuilder()
+                .setSigningKey(publicKey())
+                .build()
+                .parseClaimsJws(jwt)
+                .getBody();
+    }
+
+    @SneakyThrows
+    public PublicKey publicKey() {
```

Comment:
Method may be `private`:
```suggestion
    private PublicKey publicKey() {
```